--- /dev/null	2018-04-17 17:30:26.510053948 +0000
+++ glmark2-9c37ce30bf494c667a585c9840308e1514e23f65/Makefile	2018-05-21 02:35:54.556335094 +0000
@@ -0,0 +1,9 @@
+all: glmark2
+
+
+glmark2:
+	./waf configure --with-flavors=drm-gl,drm-glesv2,wayland-gl,wayland-glesv2,x11-gl,x11-glesv2 --data-path=/usr/share/glmark2 --prefix=/usr
+	./waf
+	
+install:
+	./waf install --destdir=$(DESTDIR)
\ No newline at end of file
--- glmark2-9c37ce30bf494c667a585c9840308e1514e23f65/waf~	2018-04-03 14:29:38.000000000 +0000
+++ glmark2-9c37ce30bf494c667a585c9840308e1514e23f65/waf	2018-05-21 02:34:28.414327290 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # encoding: ISO8859-1
 # Thomas Nagy, 2005-2016
 #
